rustic.el

```diff
@@ -75,6 +75,17 @@
                (dir (cdr (assoc-string "root" output))))
           (setq rustic--buffer-workspace dir))))))
 
+(defun rustic-buffer-project ()
```

Comment:
Yes, I found that the project root should call `cargo locate-project` too , this function fixes it.

Reply:
This function seems to return the same path as `rustic-buffer-crate`.

Reply:
Sorry for the late response, I was not sure if `rustic-buffer-crate` locates the right package root by the "Cargo.toml" file path, so I added this defun by `cargo locate`, but seems the defun `rustic-buffer-crate` works fine in my cases and I also found this far more effective than cargo locate.

Reply:
Fix by this commit: ec5fe6b81375a9f3290bf31150dc2ef5fbcc49e8